components/Dialogs/src/private/MDCAlertControllerView+Private.m

```diff
@@ -198,6 +202,11 @@ - (void)setTitleIcon:(UIImage *)titleIcon {
   [self.titleIconImageView sizeToFit];
 }
 
+- (void)setTitleIconTintColor:(UIColor *)titleIconTintColor {
+  _titleIconTintColor = titleIconTintColor;
```

Comment:
You will need to apply _titleIconTintColor to newly created titleIconImageView.
Add this in setTitleIcon:

Reply:
Good catch. Done.